dart/lib/src/sentry_envelope.dart

```diff
@@ -27,9 +38,7 @@ class SentryEnvelope {
     final newLineData = utf8.encode('\n');
     for (final item in items) {
       yield newLineData;
```

Comment:
Should this newline be added if `yield await item.envelopeItemStream();` is empty?

Reply:
not sure if I follow, you mean if the envelope has nothing?

Reply:
An envelop has multiple items. If one (attachment) item is skipped because it's file no longer exists. Should the newline before it be added? I would guess no, though I'm not sure.

Reply:
if the attachment no longer exists (content), even its item header should not be part of the envelope, the envelope should still be formed correctly otherwise it will be malformed and rejected